dotnet/src/Connectors/Connectors.MistralAI/Client/MistralPropertyType.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel.Connectors.MistralAI.Client;
+
+/// <summary>
+/// Represents the type of a MistralAI property.
+/// </summary>
+internal class MistralPropertyType
+{
+    /// <summary>
+    /// Gets or sets the type of the property.
+    /// </summary>
+    public string Type { get; set; }
+
+    /// <summary>
+    /// Gets or sets the description of the property.
+    /// </summary>
+    public string? Description { get; set; }
+
+    /// <summary>
+    /// Gets or sets the list of possible values for the property.
+    /// </summary>
+    public IList<string>? Enum { get; set; }
```

Comment:
Suggest a plural name:

`Enums`? `PossibleValues`? `ValuesDomain` ?

Reply:
This class is serialized as a schema. I'm going to replace this in a later PR.